utils/benchmark-operator.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
 install_cli() {
-  ripsaw_tmp=/tmp/ripsaw-cli
+  export UUID=${UUID:-`uuidgen`}
```

Comment:
Why are we exporting UUID again?

Reply:
Yes I could have simply done that. But by doing this way: https://github.com/cloud-bulldozer/e2e-benchmarking/pull/422/files#diff-75780417466f89cde880aac56410f858a717b735c2dfea4b4fd587ba13bb7027L4 what I am trying to achieve here is to have a same UUID across the run so that if there was a case where we end up debugging the run, using the UUID generated above we should be able to simply identify the tmp directory created for that specific run.

<a href="https://drive.google.com/file/d/1Gmeghqn385nZ2JFpJ1ufO2kQHxVCfHxm/view?usp=sharing"><img src="https://drive.google.com/file/d/1Gmeghqn385nZ2JFpJ1ufO2kQHxVCfHxm/view?usp=sharing" style="width: 500px; max-width: 100%; height: auto" title="Supporting Logs" /></a>

If you look at the screenshot here. The UUID generated in the start of the run which is due to this code change is being across everywhere throughout the run including the place where Elastic Search Results are getting published. So I felt it would be an added advantage. 

Correct me if my intuition was wrong. I am even okay to make the code change as you mentioned above if it is going to keep things simple.

Reply:
Removing the UUID in the ripsaw-cli setup step as to avoid duplicate code because every benchmark that is intended to use an UUID already generates one at the start of its run. Thanks for letting me know. Because of this discussion I have come to know that only few of our workloads actually need an UUID.